docker/Dockerfile.aarch64-linux-android

```diff
@@ -34,15 +34,21 @@ RUN /android-system.sh arm64
 
 ENV CROSS_TOOLCHAIN_PREFIX=aarch64-linux-android-
 ENV CROSS_SYSROOT=/android-ndk/sysroot
+ENV CROSS_ANDROID_SDK=$ANDROID_SDK
 COPY android-symlink.sh /
 RUN /android-symlink.sh aarch64 aarch64-linux-android
 
 COPY android-runner /
 
+COPY android.cmake /opt/android.cmake
+ENV CMAKE_TOOLCHAIN_FILE=/opt/android.cmake
+ENV CROSS_CMAKE_OBJECT_FLAGS="-DANDROID -ffunction-sections -fdata-sections -fPIC"
```

Comment:
All of the `CROSS_CMAKE_OBJECT_FLAGS` were generated manually by checking a project using CMake that failed and output the `CMAKE_C_FLAGS`, etc., and confirmed the C, C++, and ASM flags were all the same.